Podfile

```diff
@@ -47,7 +47,7 @@ target 'WordPress', :exclusive => true do
   pod 'WPMediaPicker', '~> 0.9.0'
   pod 'WordPress-iOS-Editor', '1.2'
   pod 'WordPress-iOS-Shared', '0.5.3'
-  pod 'WordPressApi', :git => "https://github.com/wordpress-mobile/WordPress-API-iOS.git"
+  pod 'WordPressApi', '~>0.4'
```

Comment:
Probably should lock it at the minor version with `'~> 0.4.0'` so it doesn't get 0.5.x for example.